packages/web-fetch/src/index.ts

```diff
@@ -0,0 +1,96 @@
+import {reatomAsync} from '@reatom/async'
+import {Atom, Ctx, isAtom, throwReatomError} from '@reatom/core'
+
+export interface ReatomFetchOptions<O, I> {
+	fetch?: (url: string, init: RequestInit) => Promise<Response>
+	init?: Omit<RequestInit, 'method' | 'signal' | 'body' | 'headers'>
+	auth?: MaybeAtom<string>
+	headers?: MaybeAtom<HeadersInit>
+	baseUrl?: string
+	method?: string
+	validate?: (o: O) => boolean | string
+	parseResponse?: (response: Response) => Promise<O>
+	serializeBody?: (i: I, headers: Headers) => BodyInit
+	validateStatus?: (status: number) => boolean | string
+}
+
+type UrlSearchParamsInit = ConstructorParameters<typeof URLSearchParams>[0]
+
+type MaybeAtom<T> = Atom<T> | T
+
+const defaults: ReatomFetchOptions<unknown, unknown> = {
```

Comment:
This is too common name, use prefix, like `createReatomFetchDefaults` or made a part of the `createReatomFetch`, like `createReatomFetch.defaults`